packages/stockflux-launcher/src/reducers/child-window/ChildWindow.js

```diff
@@ -0,0 +1,26 @@
+import childWindowState from './State';
+import actionType from './Actions';
+
+export const initialChildWindowState = {
+  window: null,
+  state: childWindowState.initial
+};
+
+export default (state, action) => {
+  switch (action.type) {
+    case actionType.changeState:
+      if (!Object.keys(childWindowState).indexOf(action.payload))
```

Comment:
I think there's a potential for a bug here. `indexOf` returns a number which refers to the position of the given parameter, which could be `0`. `-1` will be returned if the given parameter does not exist. `0` is considered a falsy value whereas `-1` is considered truthy. I assume that the intention here is to make sure that the state is one of the expected values, so it would probably be better to check explicitly for `-1` when using `indexOf` or perhaps use `.includes(...)` which reads a bit better.